src/main/java/com/spotinst/sdkjava/model/bl/ocean/aks/ImportClusterAdditionalIpConfigurationsAks.java

```diff
@@ -0,0 +1,80 @@
+package com.spotinst.sdkjava.model.bl.ocean.aks;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ImportClusterAdditionalIpConfigurationsAks {
```

Comment:
didn't find this class.

Reply:
This object is in the response, update the class name